pointercrate-user/src/error.rs

```diff
@@ -40,6 +40,15 @@ pub enum UserError {
     #[display(fmt = "No user with name {} found", user_name)]
     UserNotFoundName { user_name: String },
 
+    #[display(fmt = "No user with google account {} found", google_account_id)]
+    UserNotFoundGoogleAccount { google_account_id: String },
+
+    #[display(fmt = "The user is already linked to a Google account")]
+    AlreadyLinked,
+
+    #[display(fmt = "This operation is not applicable to this user")]
```

Comment:
Let's be a bit more descriptive and have this variant be explicitly about changing passwords on oauth accounts :)